source/MaterialXGenOgsXml/OgsXmlGenerator.cpp

```diff
@@ -137,6 +137,25 @@ namespace
         for (size_t i = 0; i < block.size(); ++i)
         {
             const ShaderPort* p = block[i];
+            if (p->getName() == "Pw")
```

Comment:
@ppenenko : unsure if documented but position *must* come first or the vertex/pixel structures will map to the wrong place as the vertex shader structure hard-codes Pm as the first struct member :(